sentry-rails/lib/sentry/rails/action_cable.rb

```diff
@@ -31,8 +31,10 @@ def capture(connection, transaction_name:, extra_context: nil, &block)
 
           def start_transaction(env, transaction_name)
             sentry_trace = env["HTTP_SENTRY_TRACE"]
+            baggage = env["HTTP_BAGGAGE"]
+
             options = { name: transaction_name, op: "rails.action_cable".freeze }
-            transaction = Sentry::Transaction.from_sentry_trace(sentry_trace, **options) if sentry_trace
+            transaction = Sentry::Transaction.from_sentry_trace(sentry_trace, baggage: baggage, **options) if sentry_trace
```

Comment:
Question: if `sentry_trace` is falsy then `transaction` is None/Null/Nil?

Reply:
yes, a new one will be created below